CMakeLists.txt

```diff
@@ -119,20 +121,11 @@ if (ANDROID)
 
   set(ANDROID_PACKAGE_SOURCE_DIR ${CMAKE_BINARY_DIR}/android-template)
 
-  include(FetchContent)
```

Comment:
This will effectively prevent fully functional qt5 builds (i.e. no more ssl on android)?

Reply:
@m-kuhn , good point. Thoughts?

Reply:
If we decide that we only want qt5 builds for "legacy reasons on dev systems" for a limited time... That's ok. We should be very clear on which ones and add an explicit note about support status for qt5 builds.

Reply:
@m-kuhn , OK, Android Qt5 support restored. IMHO, we should get rid of Qt5 build support across all platforms in one go, maybe after we do our first Qt6 release.